test/xds/xds_server_certificate_providers_test.go

```diff
@@ -269,8 +273,8 @@ func (s) TestServerSideXDS_WithValidAndInvalidSecurityConfiguration(t *testing.T
 	}
 
 	// Create an xDS-enabled grpc server that is configured to use xDS
-	// credentials, and register the test service on it. Configure a mode change
```

Comment:
instead of deleting this, you can simply edit to "and it set it on a stub server"

Reply:
can remove the below line of setting the server and starting service